src/Toolkit/Silo.php

```diff
@@ -15,19 +15,12 @@
  */
 class Silo
 {
-	/**
-	 * @var array
-	 */
 	public static $data = [];
```

Comment:
```suggestion
	public static array $data = [];
```

Reply:
I left it out again, because then PHP fails unless we also add the type to `Config::$data`. Which wouldn't be a problem for us, but in theory this could be a breaking change for others extending `Silo`. So I started to be a bit more cautious with adding typing in minor versions. What do you think?